Components/SingleArticle.jsx

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from "react"
+import { getSingleArticle } from "../api/api.js"
+
+import {useParams} from 'react-router-dom';
+
+
+export default function SingleArticle(){
+    const {article_id} = useParams();
+    const [article, setArticle] = useState([]);
```

Comment:
As a single article will be contained in a single object, I would initialise state as {}